app/Http/Traits/EnquiriesTrait.php

```diff
@@ -128,7 +131,7 @@ public function createEnquiryMessage(int $threadId, array $input): int
         return $enquiryMessage->id;
     }
 
-    public function sendEmail(string $ident, array $threadDetail, array $usersToNotify, array $jwtUser): void
+    public function sendEmail(string $ident, array $threadDetail, array $usersToNotify, array $jwtUser, string $currentUserPrefferedEmail = 'primary'): void
```

Comment:
Minor spelling error to correct:
```suggestion
    public function sendEmail(string $ident, array $threadDetail, array $usersToNotify, array $jwtUser, string $currentUserPreferredEmail = 'primary'): void
```

Reply:
updated